aptos-move/block-executor/src/view.rs

```diff
@@ -1164,7 +1166,7 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> LatestView<
         parallel_state: &ParallelState<'a, T, X>,
         delayed_write_set_ids: &HashSet<T::Identifier>,
         skip: &HashSet<T::Key>,
-    ) -> Result<BTreeMap<T::Key, (StateValueMetadata, u64)>, PanicError> {
+    ) -> PartialVMResult<BTreeMap<T::Key, (StateValueMetadata, u64)>> {
```

Comment:
why did we have to change here to PartialVMResult?

Reply:
Because caked functions can their speculative,  panic and storage errors. Mapping all of those to  panic is what the issue was